File: day2/trails/Q5.py
# write a program that takes a percentage and assigns a grade based on the following condition
# 90-100 = A
# 80-89 = B
# 70-79 = C
# 60-69 = D
# 0-59 = F

num = int(input("enter the percentage: "))

if num >= 90 and num <= 100:
    print("A")
elif num >= 80 and num <= 89:
    print("B")
elif num >= 70 and num <= 79:
    print("C")
elif num >= 60 and num <= 69:
    print("D")
else:
    print("F")

File: day2/trails/Q5.java
import java.util.Scanner;

public class Q5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Percentage: ");
        int num = sc.nextInt();

        if(num>=90 && num<=100) {
            System.out.println("Grade A");
        } else if(num>=80 && num<90) {
            System.out.println("Grade B");
        } else if(num>=70 && num<80) {
            System.out.println("Grade C");
        } else if(num>=60 && num<70) {
            System.out.println("Grade D");
        } else if(num>=50 && num<60) {
            System.out.println("Grade F");
        }else {
            System.out.println("Fail");
        }
    }
}
